src/test/scala/uk/gov/nationalarchives/tdr/api/db/repository/FileMetadataRepositorySpec.scala

```diff
@@ -285,6 +285,7 @@ class FileMetadataRepositorySpec extends TestContainerUtils with ScalaFutures wi
       )
   }
 
+  // TODO: Remove it once the refactoring is done as a port of (TDR-2477).
```

Comment:
Should re-word this comment slightly: `... as part of TDR-2477: https://national-archives.atlassian.net/browse/TDR-2477`